src/pages/index.js

```diff
@@ -59,33 +63,36 @@ const section = new Section(
       section.addItem(cardElement);
     },
   },
-  ".cards__content",
+  ".cards__content"
 );
-section.renderItems();
+//section.renderItems();
 
 Promise.all([api.getInitialCards(), api.getUserInfo()])
-//process the result
-.then(([cardData, formData]) => {
- // console.log("Card Data:", cardData);
- // console.log("Form Data:", formData)
- userinfo.setUserInfo(formData);
- userinfo.setAvatar(formData.avatar);
- cardData.forEach((item) => {
- 
-  //console.log("is like and unlike here at ForEach?",cardData)
-  //cardData.setLikeStatus(cardData.isLiked)
-  //const cardElement = createCard(item)
-   section.addItem(createCard(item));
-   //console.log("is like and unlike here at addItem???")
- })
-})
-.catch((err) => {
- console.error(err);// log the error to the console
-})
-.finally(() => {
-section.renderItems();
-//console.log("is like and unlike here at renderItems???", renderItems)
-}); 
+  //process the result
+  .then(([cardData, formData]) => {
+     console.log("Card Data:", cardData);
+     console.log("Form Data:", formData)
+    userinfo.setUserInfo(formData);
+    userinfo.setAvatar(formData.avatar);
+    cardData.forEach((item) => {
+      //console.log("is like and unlike here at ForEach?",cardData)
+      //cardData.setLikeStatus(cardData.isLiked)
+      //const cardElement = createCard(item)
+     section.addItem(createCard(item));
+     console.log("is this add Item firing", item)
+    // section.renderItems(createCard(item));
+      //console.log("is like and unlike here at addItem???")
+    }),
+      section.renderItems(cardData);
```

Comment:
`      section.renderItems(cardData);
`

This doesn't work

You need to make `renderItems` so that it could accept the array of cards